src/components/SidePanel/SidePanel.tsx

```diff
@@ -53,6 +53,11 @@ export type Props = {
    */
   pinned?: boolean;
 
+  /**
+   * Whether to show the side panel or not.
+   */
+  show?: boolean;
```

Comment:
I think the only external interface change should be a new `isAnimated` prop.

I think we should not enable animations for everyone by default. Maybe better to keep animations disabled by default. Though this could be discussed as to what is the best default.

Reply:
I think this should be wider communicated once it lands. We might want to refactor use cases of the component to use the `show` prop instead of conditional rendering in all the usages in projects. Maybe a better name for this is `isOpen`?

Reply:
And then we can have the second `isAnimated` prop, to drive that condition. I agree in the prop docs of animated we should mention that this only works with the other prop in tandem and not with conditional rendering that was previously used.